historian/src/lib.rs

```diff
@@ -1,13 +1,14 @@
 extern crate wascc_actor as actor;
 extern crate wasmdome_protocol as protocol;
+extern crate log;
```

Comment:
If you `#[macro_use]` here you don't have to explicitly import any of the macros like you do on line 11.